<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>747</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="statusL">
        <property name="text">
         <string>&lt;b&gt;Not connected&lt;/b&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="findB">
        <property name="text">
         <string>FIND</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QDateTimeEdit" name="dateTimeEdit">
        <property name="displayFormat">
         <string>dd.MM.yyyy HH:mm:ss</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="setB">
        <property name="text">
         <string>SET</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Computer</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="dayL">
             <property name="font">
              <font>
               <pointsize>32</pointsize>
              </font>
             </property>
             <property name="text">
              <string>00</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_18">
             <property name="font">
              <font>
               <pointsize>32</pointsize>
              </font>
             </property>
             <property name="text">
              <string>.</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="monL">
             <property name="font">
              <font>
               <pointsize>32</pointsize>
              </font>
             </property>
             <property name="text">
              <string>00</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_20">
             <property name="font">
              <font>
               <pointsize>32</pointsize>
              </font>
             </property>
             <property name="text">
              <string>.</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="yearL">
             <property name="font">
              <font>
               <pointsize>32</pointsize>
              </font>
             </property>
             <property name="text">
              <string>0000</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="hourL">
             <property name="font">
              <font>
               <pointsize>32</pointsize>
              </font>
             </property>
             <property name="text">
              <string>00</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_9">
             <property name="font">
              <font>
               <pointsize>32</pointsize>
              </font>
             </property>
             <property name="text">
              <string>:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="minL">
             <property name="font">
              <font>
               <pointsize>32</pointsize>
              </font>
             </property>
             <property name="text">
              <string>00</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_10">
             <property name="font">
              <font>
               <pointsize>32</pointsize>
              </font>
             </property>
             <property name="text">
              <string>:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="secL">
             <property name="font">
              <font>
               <pointsize>32</pointsize>
              </font>
             </property>
             <property name="text">
              <string>00</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="moveB">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>&gt;&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Device</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <spacer name="verticalSpacer_5">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="devDayL">
             <property name="font">
              <font>
               <pointsize>32</pointsize>
              </font>
             </property>
             <property name="text">
              <string>00</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_22">
             <property name="font">
              <font>
               <pointsize>32</pointsize>
              </font>
             </property>
             <property name="text">
              <string>.</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="devMonL">
             <property name="font">
              <font>
               <pointsize>32</pointsize>
              </font>
             </property>
             <property name="text">
              <string>00</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_24">
             <property name="font">
              <font>
               <pointsize>32</pointsize>
              </font>
             </property>
             <property name="text">
              <string>.</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="devYearL">
             <property name="font">
              <font>
               <pointsize>32</pointsize>
              </font>
             </property>
             <property name="text">
              <string>0000</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="devHourL">
             <property name="font">
              <font>
               <pointsize>32</pointsize>
              </font>
             </property>
             <property name="text">
              <string>00</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_15">
             <property name="font">
              <font>
               <pointsize>32</pointsize>
              </font>
             </property>
             <property name="text">
              <string>:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="devMinL">
             <property name="font">
              <font>
               <pointsize>32</pointsize>
              </font>
             </property>
             <property name="text">
              <string>00</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_13">
             <property name="font">
              <font>
               <pointsize>32</pointsize>
              </font>
             </property>
             <property name="text">
              <string>:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="devSecL">
             <property name="font">
              <font>
               <pointsize>32</pointsize>
              </font>
             </property>
             <property name="text">
              <string>00</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSearch_for_packages">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Search for packages</string>
   </property>
  </action>
  <action name="actionPaste_Modbus_RTU_CRC">
   <property name="text">
    <string>Calculate RTU CRC</string>
   </property>
  </action>
  <action name="actionHEX_Data_at_new_line">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>HEX Data at new line</string>
   </property>
  </action>
  <action name="actionPacket_Creator">
   <property name="text">
    <string>Packet Creator</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
